### DISCUSSION TITLE
Installation of beta?

### DISCUSSION THREAD

**Discussion by Eisbaer2** [Tasshack/dreame-vacuum#355] [Q&A] [answered]
Hello, because of my new r2312 dreame Bot I need to install the current beta in Home Assistant. I'm a Newbie @ HA - could anybody help me, how to easily install the current beta dreame-vacuum?

**Comment by Tasshack** [comment, marked as answer]
https://hacs.xyz/docs/faq/beta/

**Reply by salmundo11** [reply]
That’s 404 now.

**Reply by hockeymikey** [reply]
@salmundo11 https://web.archive.org/web/20231021001818/https://hacs.xyz/docs/faq/beta/
